src/api/Snapshot.ts

```diff
@@ -341,6 +343,39 @@ export class Snapshot extends API {
     const vp = await this.getLatestScores(space, [address])
     return Object.values(vp)[0]
   }
+
+  fetchAddressesVotesByDate = async (params: { addresses: string[] }, skip: number, batchSize: number) => {
```

Comment:
Is this written as an arrow function for some reason?

Reply:
Yes, when I tried writing it as a normal class method and calling it from the `inBatches`, the `this` didn't bind properly, so it returned an error when doing `this.fetch(...)` 🤔 

>>> Error while executing fetchAddressesVotesByDate in batches:  TypeError: Cannot read properties of undefined (reading 'fetch')